lmdeploy/pytorch/weight_loader/dist_utils.py

```diff
@@ -88,6 +116,57 @@ def colwise_parallelize_loralinear(module: torch.nn.Module,
     module._tp_mode = 'colwise'
 
 
+def _get_split_size_with_align(size: int, align: int, num_chunk: int):
+    """get split size with align."""
+    assert size % align == 0
+    num_aligned = size // align
+    split_size = _div_up(num_aligned, num_chunk) * align
+    return split_size
+
+
+def colwise_parallelize_wqlinear(mod: torch.nn.Module,
+                                 loader: ModelWeightLoader,
+                                 rank: int,
+                                 world_size: int,
+                                 prefix: str = ''):
+    """colwise parallelize wqlinear."""
+    elem_per_word = 32 // mod.w_bit
+    group_size = mod.group_size
+    lcm = math.lcm(elem_per_word, group_size)
```

Comment:
math.lcm is added with python>=3.9, failed with python3.8 https://docs.python.org/3/library/math.html#math.lcm

AttributeError: module 'math' has no attribute 'lcm'